@xen-orchestra/lite/src/App/TreeView.tsx

```diff
@@ -99,7 +99,7 @@ const TreeView = withState<State, Props, Effects, Computed, ParentState, ParentE
             id: pool.$id,
             label: (
               <span>
-                <Icon icon='cloud' /> {pool.name_label}
+                <Icon icon='warehouse' color='#0085FF' /> {pool.name_label}
```

Comment:
Why is it different from the mockup one? Did it change?

Reply:
Yes, we didn't find the same.